tests/keras/test_callbacks.py

```diff
@@ -315,7 +315,7 @@ def data_generator_graph(train):
                   metrics=['accuracy'])
 
     tsb = callbacks.TensorBoard(log_dir=filepath, histogram_freq=1,
-                                write_images=True)
+                                write_images=True, write_grads=True)
```

Comment:
Can you also add a dropout layer in the model being tested here? Since your code include handling for the learning phase.